### PR TITLE
UI for configuring logging

### PR DESCRIPTION
**Pull request by jkongie**
Adds a new card to the configurations page which contains a form to update logging configuration.

Currently the tests do not cover updating the form inputs, as I could not figure out how to make that work with Material UI/Enzyme/Jest. However, they do test that the form is submitted and the API request is made with the values in the form.

### CODE REVIEW

File: operator_ui/src/pages/Configuration/LoggingCard.test.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      sqlEnabled: false,
+    })
+
+    global.fetch.getOnce(globPath('/v2/log'), logConfig)
+    const submit = global.fetch.patchOnce(globPath('/v2/log'), logConfig)
+
+    const wrapper = mountWithProviders(<LoggingCard />)
+    await syncFetch(wrapper)
+
+    // Cannot figure out how to change the select and checkbox inputs for submit
```
Comment (jkongie) [inline operator_ui/src/pages/Configuration/LoggingCard.test.tsx:41]: @DeividasK Do you know any way to get this to work. I tried many different ways, but just couldn't get it to work

**Review by samsondav — COMMENTED** [review]
Not a review, but can we add @DeividasK as code owner for the operator_ui directory?

File: operator_ui/src/pages/Configuration/LoggingCard.test.tsx (hunk 1 above)
Comment (DeividasK) [reply operator_ui/src/pages/Configuration/LoggingCard.test.tsx:41]: I would have expected
```
wrapper.find(...).simulate('change', value)
```
to work, but I assume you've tried it already. Might also be worth trying to create a custom handler to see if that does it.

You won't be able to achieve full integration testing with DOM like interaction, but simply calling the `onChange` handler should be enough.

Comment (jkongie) [reply operator_ui/src/pages/Configuration/LoggingCard.test.tsx]: Yeah I did try to simulate and it still didn't work. Really not sure why.

I've updated the test to change the level select input with an `onChange` however I was not able to do that for the checkbox as it changed the component from a controlled input to be uncontrolled.

I think the test is OK as is and I've tested it myself in through the UI. If we really need to get this test to change the checkbox I can look at it again at a later date.

**Comment by jkongie** [comment]
> Not a review, but can we add @DeividasK as code owner for the operator_ui directory?

Yep  I've added him in

**Review by DeividasK — APPROVED** [review]

**Pull request merged**